src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -252,6 +255,23 @@ public function addRow($row)
         return $this;
     }
 
+    public function appendRow($row)
+    {
+        if (!$this->output instanceof ConsoleSectionOutput) {
+            throw new \RuntimeException(sprintf('Output should be an instance of %s when calling %s', ConsoleSectionOutput::class, __METHOD__));
```

Comment:
Missing dot at the end of the message. You should also add `"` around `%s`: `Output should be an instance of "%s" when calling "%s".`